src/kubeobjects/platform.go

```diff
@@ -1,9 +1,14 @@
 package kubeobjects
```

Comment:
there is a `kubeobjects` under `test` (for example: https://github.com/Dynatrace/dynatrace-operator/blob/03695c1d95fe519ea50492906016410452ed8926/test/helpers/kubeobjects/environment/environment.go#L14)

as these functions are only used for testing we should have it under `/test`